### DISCUSSION TITLE
Unknown database 'yourls'

### DISCUSSION THREAD

**Discussion by ansorg** [YOURLS/YOURLS#3180] [Q&A]
hi, I have yourls running via Docker: yourls-fpm-alpine + nginx + mysql containers.

By "running" I mean the webser is up and I can call /admin but I get the error 
`Unknown database 'yourls'`

Shouldn't the database be created during the initial load? 

Anything I can do to trigger the database creation?

**Comment by Akila3** [comment]
that is because you did not fill in the config.php the actual DB instance and you left it with the default YOURLS
you need to create the DB instance 1st, only then point it in your config.php file.
YOURLS doesn't create a DB for you, only populate it's stuff on existing DB
define( 'YOURLS_DB_NAME', 'put here the actual DB instance that you created in your host' );
example: define( 'YOURLS_DB_NAME', 'epiz_31420000_mydb' );

**Reply by th0rinx** [reply]
Hola ! te dejo mi guia de instalacion 

https://github.com/th0rinx/yourls-apache2
